splice_machine/src/main/java/com/splicemachine/derby/impl/sql/compile/NestedLoopJoinStrategy.java

```diff
@@ -367,53 +367,15 @@ private boolean hasJoinPredicateWithIndexKeyLookup(OptimizablePredicateList pred
      */
 
     public static double nestedLoopJoinStrategyLocalCost(CostEstimate innerCost, CostEstimate outerCost,
-                                                         double numOfJoinedRows, boolean useSparkCostFormula) {
+                                                         double numOfJoinedRows) {
```

Comment:
I think this is dangerous. I'm not too happy about the different formulas for control and Spark, but the current Spark penalty only applies in some cases, so we could end up choosing NLJ in Spark where it would be very costly.

Reply:
Right. I thought about it yesterday again for some time and I think it makes sense to revert back to two formulae. One reason is to avoid regression on OLAP, the other reason is that we probably do have special treatments for OLTP later.

Reply:
Reverted and rebased.